public/js/test/cypress/commands.js

```diff
@@ -0,0 +1,39 @@
+// ***********************************************
```

Comment:
These files are all commented out, are they templates or going to be fleshed out later?